configure.ac

```diff
@@ -15,30 +15,21 @@ AC_CONFIG_HEADER([config.h])
 
```

Comment:
The changes here are unrelated to `git.mk` and clean up the build. I would advice you to cherry-pick these changes as well.